tornjak-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tornjak-frontend",
-  "version": "0.1.0",
+  "version": "1.1.3",
```

Comment:
i don't think we should follow SPIRE versioning, at least not yet while we are still in active dev (at least shouldn't be 1.x)